pyproject.toml

```diff
@@ -20,7 +20,7 @@ classifiers = [
   "Topic :: Home Automation",
 ]
 dependencies = [
-  "aiohttp",
+  "httpx",
```

Comment:
Why switch from aiohttp to httpx?

Reply:
As far as I could tell, `aiohttp` has no out-of-the-box support for Digest. Does that sound about right?

https://github.com/aio-libs/aiohttp/issues/4939